docs/serverless/08-04-trigger-function-with-event.md

```diff
@@ -105,24 +105,52 @@ To test if the Trigger CR is properly connected to the function:
     }
     ```
 
-2. Send an event manually to trigger the function:
+2. Send an event manually to trigger the function. The event payload is delivered as the body of the HTTP request in `JSON` format. You can use [CloudEvents](https://github.com/cloudevents/spec/blob/master/spec.md) specification or the previous, Kyma-compatible solution.
```

Comment:
In case we wanna mention the used content mode (here: structured not binary), a link to the content types is nice to have: https://github.com/cloudevents/spec/blob/v1.0/http-protocol-binding.md#13-content-modes

Reply:
I rewrote it a little bit to make it more meaningful. In this tutorial, we can just stick to examples, but I would add maybe a follow-up ticket to:
1. describe events in a similar way as we had it in the Event Bus docu. (https://kyma-project.io/docs/1.9/components/event-bus/#details-event-delivery-event-payload). 
2. Maybe we can add some info on how this compatibility layer works, compare it to how the events are processed now - for the people who still use ut What do you think? 
I think we still need this info, enriched with CloudEvents-related data.